FirebaseSessions/Sources/GoogleDataTransport+GoogleDataTransportProtocol.swift

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+@_implementationOnly import GoogleDataTransport
+
+struct GoogleDataTransportProtocolErrors {
+  static let writeFailureError = NSError(domain: "GoogleDataTransportProtocol", code: -1)
+}
```

Comment:
Did you consider a Swift error enum? https://docs.swift.org/swift-book/LanguageGuide/ErrorHandling.html

Reply:
That's much better